wolfssl/wolfcrypt/sp_int.h

```diff
@@ -739,10 +741,21 @@ typedef struct sp_ecc_ctx {
  * completion.
  */
 #define FP_WOULDBLOCK   (-4)
+#define MP_WOULDBLOCK   (-4)
 /* Unused error. Defined for backward compatibility. */
 #define MP_NOT_INF      (-5)
-/* Unused error. Defined for backward compatibility. */
-#define MP_RANGE        MP_NOT_INF
+/* hardware error, consider falling back to SW */
+#define MP_HW_ERROR     (-6)
```

Comment:
Lets use the existing WC_HW_E

Reply:
Debug logs with WOLFSSL_MSG could be used to narrow down location of hardware error and distinguish between math hardware acceleration and others if wanted.